research/conditional/train/cc_train.py

```diff
@@ -280,7 +353,17 @@ def main(
         for name, param in model.named_parameters():
             print(name, param.shape)
 
-    param_grops, ratios_in_group_order = make_param_groups_and_lr_ratios(args, model)
+    # set args for optimizer param_groups
+    baseline_args = [args.learning_rate, args.final_lr_fraction]
+    baseline_keys = [
+        "ratios_lr_in_group_order",
+        "scheduler_fractions_in_group_order",
+    ]  # key names like arguments in get_scheduler :), best idea i had :(
+    relative_params_all = [args.relative_lr, args.relative_scheduler_fraction]
```

Comment:
nie no w sumie plz przezwijmy to relative_scheduler_fraction bo ten kod jest niezrozumiały.
Dosłownie coś w stylu "final_lr_fraction"

Reply:
done